src/OpenTelemetry/Metrics/Processors/PushMetricProcessor.cs

```diff
@@ -61,6 +61,7 @@ protected override void Dispose(bool disposing)
                 {
                     this.token.Cancel();
                     this.exporter.Dispose();
+                    this.exportTask.Wait();
```

Comment:
Btw, do we see the real difference between push/pull metric processors? I personally don't (and I'm thinking maybe we should remove push/pull concept from the draft spec).

Reply:
Not much, since the MeterProvider provided a way to "Collect". PushProcessor calls this Collect every N seconds. PullProcessor calls this Collect whenever its asked to.